lenses/logrotate.aug

```diff
@@ -19,7 +19,7 @@ module Logrotate =
    let eol = Util.eol
    let num = Rx.relinteger
    let word = /[^,#= \n\t{}]+/
-   let filename = /\/[^,#= \n\t{}]+/
+   let filename = /(\/[^,#= \n\t{}]+|"\/[^,#= \n\t{}]+")/
```

Comment:
You could also use `Quote.do_dquote_opt`:

```suggestion
let filename = Quote.do_quote_opt (store /\/[^"',#= \n\t{}]+/)
```

and change `store filename` in `rule` to `filename`

Reply:
Thank you for this hint - this looks better. Added the proposal and fixed the test (the proposed variant does not show quotes as value).